content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -39,10 +46,11 @@ trusts. For example, you would do this with a golang TLS config by parsing the
 certificate chain and adding the parsed certificates to the `RootCAs` field
 in the [`tls.Config`](https://godoc.org/crypto/tls#Config) struct.
 
-The CA certificate bundle is automatically mounted into pods using the default
-service account at the path `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`.
-If you are not using the default service account, ask a cluster administrator to
-build a configmap containing the certificate bundle that you have access to use.
+The cluster root CA certificate bundle is automatically mounted into pods using
+the default service account at the path
+`/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`.  If you are not using
+the default service account, ask a cluster administrator to build a configmap
```

Comment:
Does this help a reader who is their own cluster operator? If not, is there a way to signpost that reader to learn about making that ConfigMap?

(If there's no existing document then, IMO, it's OK to track that in a new issue and leave things there).